### PR TITLE
Handle not available vor_update_timestamp

### PR DESCRIPTION
**Pull request by HazalCiplak**
Removed assertion error for not available vor update date instead we will not populate that field until it is available

https://github.com/elifesciences/data-hub-issues/issues/943

### CODE REVIEW

File: tests/unit_tests/docmaps/v2/codecs/docmaps_test.py [hunk 1]
```diff
@@ -166,6 +166,18 @@ def test_should_populate_assertion_status_after_first_version_as_corrected_with_
...
+        assertion_list = get_docmap_assertions_for_vor_steps(
+            query_result_item=DOCMAPS_QUERY_RESULT_ITEM_WITH_VOR_VERSIONS_2,
+            manuscript_version=MANUSCRIPT_VERSION_WITH_EVALUATIONS_1,
+            vor_version_number=VOR_VERSIONS_2['vor_version_number'],
+            vor_updated_timestamp=None
+        )
+        assertion_status = assertion_list[0]['status']
+        assertion_happened = assertion_list[0]['happened']
+        assert assertion_status == 'corrected'
+        assert not assertion_happened
```
Comment (de-code) [inline tests/unit_tests/docmaps/v2/codecs/docmaps_test.py]: more like a question, would it be better to test that it is None?

**Review by de-code — APPROVED** [review]
